lib/gui/app/app.js

```diff
@@ -284,6 +285,23 @@ app.run(($window) => {
       popupExists = false
     }).catch(exceptionReporter.report)
   })
+
+  /**
+   * @summary Helper fn for events
+   * @function
+   * @private
+   * @example
+   * window.addEventListener('click', extendLock)
+   */
+  const extendLock = () => {
+    updateLock.extend()
+  }
+
+  $window.addEventListener('click', extendLock)
+  $window.addEventListener('touchstart', extendLock)
```

Comment:
It probably makes more sense to use `touchend` here given that a user could _in theory_ pretend Etcher Pro is a pet.